cmd/keymasterd/storage.go

```diff
@@ -22,6 +22,15 @@ const userProfileSuffix = ".gob"
 const profileDBFilename = "userProfiles.sqlite3"
 const cachedDBFilename = "cachedDB.sqlite3"
 
+func (config *ProfileStorageConfig) applyDefaults() {
+	if config.SyncDelay < 1 {
+		config.SyncDelay = time.Second * 3
+	}
+	if config.SyncInterval < 1 {
+		config.SyncInterval = time.Minute * 5
+	}
+}
+
```

Comment:
The location of this file is suspect. It seems like this should be part of the config.go, and this should be applied there, at config load time, on empty fields

Reply:
I did it that way the first time around, but that tests failing because the sync loop would spam the test logs, overflowing the TravisCI limits. It's because initDB() is called from all the tests. It seemed ridiculous to modify all the tests to add the call to applyDefaults().

An alternative would be to not run the sync goroutine if the interval is zero (i.e. uninitialised).